### PR TITLE
Add Python 3 support + remove scanning stuff

### PR DESCRIPTION
**Pull request by sn0opy**
This PR will make RM compatible with Python 3. Unfortunately, this also drops Python 2 support completely. Only tested with Python 3.7.3 on an existing DB. 

Things to consider:
* Python 3.6 is completely untested
* I've added a verison check to force the python version to at least 3.6
* ~@delenca's [MrMime](https://github.com/delenca/MrMime) fork is being used. Should probably reforked with @Map-A-Droid for obvious reasons~ ✅fixed
* ~I haven't used a clean DB setup. So, before merging this, one should test to run the setup process to create a db schema~ ✅fixed

### CODE REVIEW

File: templates/map.html [hunk 1]
```diff
@@ -365,82 +365,6 @@ <h3>Weather Alerts</h3>
...
-                <input id="geoloc-switch" type="checkbox" name="geoloc-switch" class="onoffswitch-checkbox" checked/>
-                <label class="onoffswitch-label" for="geoloc-switch">
-                <span class="switch-label" data-on="On" data-off="Off"></span>
-                <span class="switch-handle"></span>
-                </label>
-              </div>
-            </div>
-              {% endif %}
-            <div class="form-control switch-container" >
-              <h3>Start map at my position</h3>
```
Comment (na-ji) [inline templates/map.html]: `Start map at my position` and `Follow me` might be used today. Wouldn't it be better to keep them?

Comment (sn0opy) [reply templates/map.html]: I have to admit. I've tested it and it didn't work for me. Even on https with location API being enabled. I'll have another look into it.

Comment (na-ji) [reply templates/map.html]: I don't know what are the conditions to make it work, but on my instance it works. Do you want me to send you the URL?

File: config/config.ini.example [hunk 2]
```diff
@@ -85,42 +84,6 @@
...
-#use-altitude-cache             # Query the Elevation API for each step, rather than only once, and store results in the database.
-                                # Make sure your Google Elevation API is enabled
-#workers-per-hive:              # Only referenced when using --beehive. Sets number of workers per hive. (default=1)
-#workers:                       # Number of search worker threads to start. (default=#accounts)
-#spawn-delay:                   # Number of seconds after spawn time to wait before scanning to be sure the Pokemon is there. (default=10)
-#kph:                           # Set a maximum speed in km/hour for scanner movement. 0 to disable.  (default=35)
-#bad-scan-retry:                # Number of bad scans before giving up on a step. (default=2, 0 to disable)
-#skip-empty                     # Enables skipping of empty cells in normal scans - requires previously populated database. (not to be used with -ss)
-#min-seconds-left:              # Time that must be left on a spawn before considering it too late and skipping it. (default=0)
-#gym-info                       # Enables detailed gym info collection. (default=False)
```
Comment (daangroot) [inline config/config.ini.example]: This argument is used by the front end, so it should not be removed. It enables the gym sidebar.

Comment (sn0opy) [reply config/config.ini.example]: `gym_info` should not be used for this I think. I'd rather introduce a new parameter to  enable the sidebar, or just show the settings permanently. This paramter has been used in the search worker only. There's no other place wher it is being used.

File: pogom/utils.py [hunk 3]
```diff
@@ -966,8 +957,9 @@ def is_imagemagick_binary(binary):
         process = subprocess.Popen([binary, '-version'],
                                    stdout=subprocess.PIPE)
         out, err = process.communicate()
-        return "ImageMagick" in out
-    except Exception:
+        return "ImageMagick" in out.decode('utf8')
+    except Exception as e:
+        print(e)
```
Comment (na-ji) [inline pogom/utils.py]: This is printing a warning on every startup

Comment (sn0opy) [reply pogom/utils.py]: Please post that warning

Comment (na-ji) [reply pogom/utils.py]:
```
[Errno 2] No such file or directory: 'magick': 'magick'
```
The function is checking two binaries for imagemagick. Depending on your OS, you will always have this exception printed.

Comment (sn0opy) [reply pogom/utils.py]: Now I understand, what the problem was... fixed.

**Pull request merged**